src/routes/docs/products/functions/develop-locally/+page.markdoc

```diff
@@ -21,9 +21,13 @@ We use Docker to replicate the production environment for the local deployment o
 
 Use the `appwrite run functions` command to develop your function locally.
 
+| parameter       | Description                                                                                                |
+|-----------------|------------------------------------------------------------------------------------------------------------|
+| `--port`        | You can set your function port; it defaults to `3000`, or the closest available, i.e. `3001`, `3002`, etc. |
+| `--function-id` | You can autoselect a function, so you don't have to click through the list each time.                      |
```

Comment:
Let's get rid of the "You can" to make it concise and more assertive.

> You can autoselect a function

You're not really auto-selecting because this parameter allows you to manually select the function ahead of time rather than selecting it interactively